Description: Invalid Code-block style for the SPcod and SPcoc parameters
Author: Mathieu Malaterre <malat@debian.org>
Bug-Debian: http://bugs.debian.org/668629

--- a/appsrc/misc/jpegdump.cc
+++ b/appsrc/misc/jpegdump.cc
@@ -820,12 +820,12 @@ public:
 			
 			CodeBlockStyle				= buffer[8];
 			// Table A.19 - Code-block style for the SPcod and SPcoc parameters 
-			SelectiveArithmeticCodingBypass                 = (Scod & 0x01) != 0;
-			ResetContextProbabilitiesOnCodingPassBoundaries = (Scod & 0x02) != 0;
-			TerminationOnEachCodingPass                     = (Scod & 0x04) != 0;
-			VerticallyCausalContext                         = (Scod & 0x08) != 0;
-			PredictableTermination                          = (Scod & 0x10) != 0;
-			SegmentationSymbolsAreUsed                      = (Scod & 0x20) != 0;
+			SelectiveArithmeticCodingBypass                 = (CodeBlockStyle & 0x01) != 0;
+			ResetContextProbabilitiesOnCodingPassBoundaries = (CodeBlockStyle & 0x02) != 0;
+			TerminationOnEachCodingPass                     = (CodeBlockStyle & 0x04) != 0;
+			VerticallyCausalContext                         = (CodeBlockStyle & 0x08) != 0;
+			PredictableTermination                          = (CodeBlockStyle & 0x10) != 0;
+			SegmentationSymbolsAreUsed                      = (CodeBlockStyle & 0x20) != 0;
 
 			WaveletTransformation = buffer[9];
 			descriptionOfWaveletTransformation = getDescriptionOfWaveletTransformation(WaveletTransformation);
